src/main/resources/hudson/plugins/ec2/EC2Cloud/help-sshKeysCredentialsId.html

```diff
@@ -22,14 +22,21 @@
 THE SOFTWARE.
 -->
 <div>
-		A credential of the type "SSH Username with private key" with the RSA
-		private key of the EC2 key pair. You can use the AWS Console
-		or any other tool of your choice to generate the key pair.
-		The username is used, and the credential should not have a passphrase.
-
-		<p>
-		An RSA private key for EC2 is a text
-		that starts with "-----BEGIN RSA PRIVATE KEY-----". You normally
-		generate it via using either the AWS Management Console, the "ec2-add-keypair" tool or a GUI tool
-		like ElastiFox.
+	<p>If no credential (ie. "none") is selected then dynamically managed SSH
+	key pairs will be used.  For each instance that is requested a new EC2 key pair
+	will be created and used only for the lifespan of each node.  Upon termination,
+	the keypair will then be removed from EC2.</p>
+
+
+	<p>Optionally, you may specify a Jenkins credential of the type "SSH Username with
+	private key" containing the private key of the EC2 key pair. You can use the AWS Console
+	or any other tool of your choice to generate the key pair.</p>
+
+
+	<p>The username is not used, and the credential should not have a passphrase.</p>
+
+
+	<p>An RSA private key for EC2 is a text that starts with
+	"-----BEGIN RSA PRIVATE KEY-----". You normally generate it via using either
+	the AWS Management Console, the "ec2-add-keypair" tool or a GUI tool like ElastiFox.</p>
```

Comment:
BTW the reformatting makes the diff harder to read: obscures the fact that you are not actually editing this paragraph.